resteasy-cdi/pom.xml

```diff
@@ -61,6 +61,12 @@
             <artifactId>jboss-logging-processor</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.jboss.weld</groupId>
+            <artifactId>weld-api</artifactId>
+            <version>3.1.Final</version>

```

Comment:
You should avoid setting the version here and let it be driven by the dependencies bom (which I believe already includes the org.jboss.weld:weld-api dependency).